### ISSUE TITLE
Changing galaxy directory path when using self-installed conda

### ISSUE THREAD

**Issue by ppericard** [galaxyproject/galaxy#9813]
I installed a fresh Galaxy 20.01 from the Github repo on a new VM. I don't have conda installed on that VM so Galaxy self-installed conda at startup in `/media/data/galaxy/database/dependencies/_conda`.

I'm now trying to use this self-installed conda to manually create environments for some tools I'm currently developing that don't have all their dependencies available on conda.

However, before that, I moved/renamed the galaxy directory to `/media/data/0_galaxy`.

The pb is I can't use the self-installed conda now because all the conda paths are hardcoded as they were when first installed.

e.g. `/media/data/0_galaxy/database/dependencies/_conda/bin/conda`:
```
#!/media/data/galaxy/database/dependencies/_conda/bin/python
# -*- coding: utf-8 -*-
import sys
# Before any more imports, leave cwd out of sys.path for internal 'conda shell.*' commands.
# see https://github.com/conda/conda/issues/6549
if len(sys.argv) > 1 and sys.argv[1].startswith('shell.') and sys.path and sys.path[0] == '':
    # The standard first entry in sys.path is an empty string,
    # and os.path.abspath('') expands to os.getcwd().
    del sys.path[0]

if __name__ == '__main__':
    from conda.cli import main
    sys.exit(main())
```

You can see in this file that the hashbang path is hardcoded to the previous PATH.

I have a similar problem with tools installed from the Toolshed after moving/renaming the galaxy directory. Galaxy created new directories using the original path to store these tools.

So, basically, is there a way to reset Galaxy to use a new path, or is the galaxy path completely hardcoded and the directory cannot be moved or renamed ?

Thanks in advance for you help.
Cheers. Pierre

**Comment by nsoranzo** [comment]
Sorry, that's how conda works and there's nothing we can do about that. See e.g. https://github.com/conda/conda/issues/3097
